kibana-reports/public/components/context_menu/context_menu.js

```diff
@@ -1,3 +1,22 @@
+/*
```

Comment:
this is the elasticsearch license, looks like it's accidentally added to some files, ESLint problem?

Reply:
That's interesting, no idea how it gets added.